### PR TITLE
Register page

### PR DESCRIPTION
**Pull request by shraddha761**
## Related Issue

Closes: #440 

## Checklist:

- [x] My code adheres to the established style guidelines of this project.
- [x] I have conducted a self-review of my code.
- [x] I have included comments in areas that may be difficult to understand.
- [x] I have made corresponding updates to the project documentation.
- [x] My changes have not introduced any new warnings.

## Screenshots

|           Original           |           Updated           |
| :--------------------------: | :-------------------------: |
| Original screenshot | Updated screenshot |

Please provide any necessary screenshots to illustrate the changes made.

### CODE REVIEW

**Comment by shraddha761** [comment]
@anmode please review

**Comment by anmode** [comment]
I did not like the designing... Please see: https://github.com/anmode/grabtern-frontend/pull/542
and the register page is not optimized for mobile too

**Comment by shraddha761** [comment]
@anmode Yahh IK Design is not so good now

**Comment by shraddha761** [comment]
@anmode please review now

**Comment by anmode** [comment]
Cool, Some fixes:
1. Please use UI button as register button is of old UI, you can see components/ui/button
2. optimize mobile view, reduce side padding as form looking so compact and a `or` line extending the form ( a bug)

**Comment by anmode** [comment]
kch changes kiye... @shraddha761 please update your branch with develop

**Comment by shraddha761** [comment]
@anmode Hn maine kre

**Comment by anmode** [comment]
please fix the test:
```
./components/layout/Header.js:14:9
Syntax error: Identifier 'ButtonLink' has already been declared.

  12 | import clsx from "clsx";
  13 | import { encryptData, decryptData } from "../../hook/encryptDecrypt";
> 14 | import { ButtonLink } from "../UI";
     |          ^
  15 |
  16 | function Header() {
  17 |   const {


```

File: components/layout/Header.js [hunk 1]
```diff
@@ -11,6 +11,7 @@ import { useAuth } from "../../context/AuthContext";
 import { ButtonLink } from "../UI";
 import clsx from "clsx";
 import { encryptData, decryptData } from "../../hook/encryptDecrypt";
+import { ButtonLink } from "../UI";
```
Comment (anmode) [inline components/layout/Header.js:14]: remove it

**Review by anmode — COMMENTED** [review]
simple

**Comment by anmode** [comment]
`npx prettier --write .`

**Comment by anmode** [comment]
Screenshot 2023-07-21 at 10 30 45 PM
 
please reduce the padding.

**Comment by anmode** [comment]
Screenshot 2023-07-21 at 11 21 02 PM
text align :(

File: components/layout/Header.js [hunk 2]
```diff
@@ -131,7 +130,7 @@ function Header() {
 
             {/* signup button appear only if no user logged in*/}
             {!(isUserLoggedIn || isMentorLoggedIn) && (
-              <ButtonLink text="Sign Up" href="/auth/register" />
+              <ButtonLink text="Sign Up" href="/register" />
```
Comment (anmode) [inline components/layout/Header.js:133]: revert please it is `auth/register` only

**Review by anmode — CHANGES_REQUESTED** [review]
last

**Comment by shraddha761** [comment]
@anmode done with changes

**Review by anmode — APPROVED** [review]
Perfect! Congrats.

**Comment by shraddha761** [comment]
> Perfect! Congrats.

thank you

**Pull request merged**